src/scripts/main.js

```diff
@@ -1,3 +1,81 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('.field');
+const rowButtons = document.querySelectorAll('[class*="row"]');
+const columnButtons = document.querySelectorAll('[class*="column"]');
+const rows = [...table.querySelectorAll('tr')];
+const column = rows[0].cells;
+let rowsNumber = rows.length;
+let columnNumber = column.length;
+
+document.body.addEventListener('click', (e) => {
+  const button = e.target.closest('.button');
+
+  if (button) {
+    const buttonFunc = [...button.classList];
+    const regex = /column/g;
+    const containColumns = buttonFunc.some((cls) => regex.test(cls));
+
+    table.innerHTML = '';
```

Comment:
Clearing innerHTML of the table will remove all the rows and cells. Make sure that this is the intended behavior as it can be expensive in terms of performance if the table is large. Consider using a more efficient way to update the DOM if necessary.